README.md

```diff
@@ -23,7 +23,17 @@ Get up and running in a few minutes with our drop-in turn-by-turn navigation `Na
 
 ## Installation
 
-To install Mapbox Navigation using [Carthage](https://github.com/Carthage/Carthage/) v0.19.0 or above:
+To install Mapbox Navigation using [CocoaPods](https://cocoapods.org/):
+
+1. Specify the following dependency in your Podfile:
+   ```ruby
+   pod 'MapboxNavigation', '~> 0.4.0'
```

Comment:
Not since we published to the CocoaPods central repo and the UI podspec specifies
```ruby
s.dependency "MapboxCoreNavigation", "#{s.version.to_s}"
```